packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1092,7 +1099,6 @@ export default class Meetings extends WebexPlugin {
       meetingInfo: cloneDeep(meeting.meetingInfo),
       locusInfo: {
         // locusInfo can be quite big, so keep just the minimal info
-        sequence: meeting.locusInfo?.sequence,
```

Comment:
I've removed the sequence, because it's not really needed, I only added it in the first place, because I thought it was a simple string and I thought that maybe it will be useful in the future, more for debugging then actual metrics, but it turns out that sequence is an object that can contain also arrays so I don't want to be bothered with storing it.